imagestreams/nodejs-rhel.json

```diff
@@ -0,0 +1,147 @@
+{
+  "kind": "ImageStream",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "nodejs",
+    "annotations": {
+      "openshift.io/display-name": "Node.js"
+    }
+  },
+  "spec": {
+    "tags": [
+      {
+        "name": "latest",
+        "annotations": {
+          "openshift.io/display-name": "Node.js (Latest)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Build and run Node.js applications on UBI. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container/blob/master/12/README.md.\n\nWARNING: By selecting this tag, your application will automatically update to use the latest version of Node.js available on OpenShift, including major version updates.",
+          "iconClass": "icon-nodejs",
+          "tags": "builder,nodejs",
+          "supports":"nodejs",
+          "sampleRepo": "https://github.com/sclorg/nodejs-ex.git"
+        },
+        "from": {
+          "kind": "ImageStreamTag",
+          "name": "12-el8"
+        },
+        "referencePolicy": {
+          "type": "Local"
+        }
+      },
+      {
+        "name": "12-ubi8",
+        "annotations": {
+          "openshift.io/display-name": "Node.js 12 (UBI 8)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Build and run Node.js 12 applications on UBI 8. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container/blob/master/12/README.md.",
+          "iconClass": "icon-nodejs",
+          "tags": "builder,nodejs",
+          "version": "12",
+          "sampleRepo": "https://github.com/sclorg/nodejs-ex.git"
+        },
+        "from": {
+          "kind": "DockerImage",
+          "name": "registry.redhat.io/ubi8/nodejs-12:latest"
+        },
+        "referencePolicy": {
+          "type": "Local"
+        }
+      },
+      {
+        "name": "12-el7",
+        "annotations": {
+          "openshift.io/display-name": "Node.js 12 (RHEL 7)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Build and run Node.js 12 applications on RHEL 7. For more information about using this builder image, including OpenShift considerations, see https://github.com/sclorg/s2i-nodejs-container/blob/master/12/README.md.",
+          "iconClass": "icon-nodejs",
+          "tags": "builder,nodejs",
+          "version": "12",
+          "sampleRepo": "https://github.com/sclorg/nodejs-ex.git"
+        },
+        "from": {
+          "kind": "DockerImage",
+          "name": "registry.redhat.io/rhscl/nodejs-12-rhel7:latest"
+        },
+        "referencePolicy": {
+          "type": "Local"
+        }
+      },
+      {
+        "name": "12",
```

Comment:
Shouldn't this tag rather link to `12-ubi7 (`"kind": "ImageStreamTag",`) to not duplicate data if possible?

Reply:
I considered it, it would be cleaner, but the OpenShift documentation seems to imply that using ImageStreamTag makes it "floating" (iow possibly subject to future change).  @gabemontero any thoughts here?

Reply:
you could have both a a 12-ubi7 and 12 tag pointing to the same image

12 is needed for backward compatibility since that 10 already exists ... 12-ubi7 could be the "new form"

And then when a new image comes around after this ... say v16 ... it can just be 16-ubiwhatever

Reply:
Based on that, I left this as it was, let me know if you really think otherwise.

Reply:
Ok, I'm fine with how it is now.